src/views/catalog/templatescatalog/components/TemplatesCatalogDrawer/hooks/useCreateDrawerForm.tsx

```diff
@@ -212,6 +221,15 @@ const useCreateDrawerForm = (
         vmDraft.spec.template.spec.domain.cpu.cores = cpu?.cores;
         vmDraft.spec.template.spec.domain.memory.guest = memory;
 
+        if ('running' in vmDraft?.spec) {
+          if (vmDraft.spec.running) {
+            vmDraft.spec.runStrategy = 'Halted';
+          } else {
+            vmDraft.spec.runStrategy = 'Always';
+          }
+          delete vmDraft.spec.running;
+        }
+
```

Comment:
```suggestion
        if ('running' in vmDraft?.spec) {
            vmDraft.spec.runStrategy = vmDraft.spec.running ? 'Always' : 'Halted';
          delete vmDraft.spec.running;
        }

```

Reply:
here u are doing running === 'Halted' , but in other places its 'Always', so changed it here also to Always, please apply to the another location

Reply:
done.